tests/dualtor/test_tor_ecn.py

```diff
@@ -57,12 +59,12 @@ def build_encapsulated_ip_packet(
     server_ipv4 = server_ips["server_ipv4"].split("/")[0]
     config_facts = tor.get_running_config_facts()
     try:
-        peer_ipv4_address = [dut_name["address_ipv4"] for dut_name in config_facts["PEER_SWITCH"].values()][0]
+        peer_ipv4_address = [_["address_ipv4"] for _ in config_facts["PEER_SWITCH"].values()][0]
     except IndexError:
         raise ValueError("Failed to get peer ToR address from CONFIG_DB")
 
-    tor_ipv4_address = [addr for addr in config_facts["LOOPBACK_INTERFACE"]["Loopback0"]
-                        if is_ipv4_address(addr.split("/")[0])][0]
+    tor_ipv4_address = [_ for _ in config_facts["LOOPBACK_INTERFACE"]["Loopback0"]
```

Comment:
Suggest using meaningful name here since we are taking the value directly

Reply:
I had issue in giving name addr. If i keep using _ then it is fine.

IndexError('list index out of range',)
Traceback (most recent call last):
  File "/var/src/sonic-mgmt-int/tests/common/plugins/log_section_start/__init__.py", line 60, in _fixture_func_decorator
    return fixture_func(*args, **kargs)
  File "/var/src/sonic-mgmt-int/tests/dualtor/test_tor_ecn.py", line 132, in build_non_encapsulated_ip_packet
    if is_ipv4_address(_.split("/")[0])][0]
IndexError: list index out of range

Reply:
Looks like you may be missing an `addr` before the call to `split()` in the snippet that you pasted - if you fix that does it still give you an error?

Reply:
It fixes. thanks.